components/ILIAS/Scorm2004/classes/class.ilSCORM13PlayerGUI.php

```diff
@@ -487,7 +490,7 @@ public function getPlayer(): void
         if ($this->slm->getCacheDeactivated()) {
             $this->tpl->setVariable('JS_SCRIPTS', 'ilias.php?baseClass=ilSAHSPresentationGUI' . '&cmd=getRTEjs&ref_id=' . $this->ref_id);
         } else {
-            $this->tpl->setVariable('JS_SCRIPTS', '../components/ILIAS/Scorm2004/scripts/buildrte/rte-min.js');
+            $this->tpl->setVariable('JS_SCRIPTS', 'components/ILIAS/Scorm2004/scripts/buildrte/rte-min.js');
```

Comment:
IMO this will not work, since the browser will not have access to this script in ILIAS 10.x.